components/cards/ActionForm.tsx

```diff
@@ -298,7 +299,9 @@ export function ActionForm(props: ActionFormProps) {
                 <InfoTooltip
                   content={
                     <div style={{ textAlign: "left" }}>
-                      Be honest about your personal valuation!
+                      Be honest about your personal valuation! A{" "}
+                      {ethers.utils.formatEther(minForSalePrice)} ETHx minimum
+                      valuation is enforced to prevent low-value squatting.
                       <br />
                       <br />
                       You'll have 7 days to accept or reject any bid that meets
```

Comment:
Can we change the text here to `You'll have 7 days to accept or reject any bid that meets your price.`?

The change here is just from "this" to "your" price.

Reply:
Thanks!